deploy/resources/addons/02-clustertasks/git-clone/git-clone-v0-16-3-task.yaml

```diff
@@ -69,13 +69,26 @@ spec:
       description: git no proxy - opt out of proxying HTTP/HTTPS requests
       type: string
       default: ""
+    - name: verbose
+      description: log the commands used during execution
+      type: string
+      default: "true"
   results:
     - name: commit
       description: The precise commit SHA that was fetched by this Task
+    - name: url
+      description: The precise URL that was fetched by this Task
   steps:
     - name: clone
-      image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:v0.14.2
+      image: registry.redhat.io/openshift-pipelines-tech-preview/pipelines-git-init-rhel8:1.1
```

Comment:
So… this will need to refer to 1.2 at the time of the release, wouldn't they ?

Reply:
Yes, when we create a new branch v1.2, there we will refer to that. We will keep refering to as first as 1.2rc1, then 1.2.

Reply:
hum, would there be a way to automate this in the future ? (aka keeping the *upstream* reference and doing a search-and-replace as we do for the image referenced in `controller.yaml`)
(no need to do that now, just thinking :wink: )

Reply:
this step is handled by csv, we don't do any change here, this image will not be able until release, so we cant do testing.